Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,24 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    expand = File.expand_path("~/Applications")
+    File.exist?("#{expand}/Firefox.app")
```

Comment:
@skycaptain Fair point. It doesn't need to stretch to two lines. Thanks for the suggestion :+1:.